shared/oae/api/oae.api.util.js

```diff
@@ -1866,7 +1866,5 @@ define(['exports', 'require', 'jquery', 'underscore', 'oae.api.config', 'markdow
      *
      * @return {String}   Event(s) that can fire when the content of an input field changes
      */
-    var getFormChangeEventNames = exports.getFormChangeEventNames = function() {
-        return $('html').hasClass('ie-lt10') ? 'change keyup paste cut' : 'change input';
-    };
+    var getFormChangeEventNames = exports.getFormChangeEventNames = return "change input";
```

Comment:
About this file: I didn't find any references for the function `getFormChangeEventNames` to replace.